test/keyboard.spec.ts

```diff
@@ -14,12 +14,16 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import utils from './utils';
 
-const utils = require('./utils');
-const {FFOX, CHROMIUM, WEBKIT, MAC, LINUX} = require('./utils').testOptions(browserType);
+declare let events: string[];
```

Comment:
Oh this is ugly :( Should we relax our config for the tests?

Reply:
Our tests access globals inside of the page. TypeScript complains of accessing globals that it doesn't know about. It instead lets us do window['anything'], or declare vars at the top. I'm fine with picking one convention and sticking with it. I don't know of any typescript config that allows unknown global variables.

Reply:
- I can put the offending variables into types.d.ts.
- Use `window['somevar']` syntax everywhere
- Put `declare let somevar` at the top of files where `somevar` is used
- Create a Page Object Model for textarea.html and other offending files, remove global variables

Reply:
- Exploring POM sounds interesting.
- Can we at least declare them as properties on Window, and access through window.somevar? These globals declared in the Node are confusing. We should figure out something that we can suggest to our users.

Reply:
There isn't a way to have them on the window object but not in the global scope.

If our users have global variables that they are accessing from within `page.evalaute`, they should already be declaring them in some .d.ts or .ts global namespace. I'll put them in our test/types.d.ts to match that. If it's ugly thats a good sign that we and our users should build POM.